src/systems/systems/src/apu/APS3200.md

```diff
@@ -0,0 +1,212 @@
+Curve fitted polynomial regression generation: http://polynomialregression.drque.net/online.php
+
+Data points, based on a video by Komp with OAT of ~5 degrees. Note that OAT isn't yet part of the formula,
+as we don't know the exact temperature effects on EGT yet (linear or not?). I also suspect there is an altitude
+EGT effect (less air density causing higher EGT during start as it provides less cooling?), but that effect is also unknown.
+
+# N to EGT
+
+```
+N, EGT
+1, 0;
+2, 0;
+3, 0;
+4, 0;
+5, 0;
+6, 0;
+7, 0;
+8, 0;
+9, 0;
+10, 0;
+11, 10;
+12, 40;
+13, 90;
+14, 130;
+15, 165;
+16, 200;
+17, 250;
+18, 310;
+19, 330;
+20, 390;
+21, 440;
+22, 470;
+23, 505;
+24, 540;
+25, 555;
+26, 585;
+27, 615;
+28, 635;
+29, 650;
+30, 665;
+31, 670;
+32, 675;
+33, 680;
+34, 685;
+35, 695;
+36, 700;
+37, 705;
+38, 710;
+39, 715;
+40, 715;
+41, 715;
+42, 720;
+43, 720;
+44, 720;
+45, 715;
+46, 715;
+47, 715;
+48, 710;
+49, 700;
+50, 700;
+51, 695;
+52, 690;
+53, 685;
+54, 680;
+55, 675;
+56, 665;
+57, 660;
+58, 655;
+59, 655;
+60, 650;
+61, 645;
+62, 640;
+63, 640;
+64, 635;
+65, 630;
+66, 625;
+67, 615;
+68, 605;
+69, 595;
+70, 590;
+71, 575;
+72, 570;
+73, 555;
+74, 535;
+75, 525;
+76, 520;
+77, 515;
+78, 505;
+79, 500;
+80, 490;
+81, 485;
+82, 475;
+83, 470;
+84, 460;
+85, 455;
+86, 455;
+87, 450;
+88, 440;
+89, 435;
+90, 430;
+91, 430;
+92, 420;
+93, 420;
+94, 415;
+95, 415;
+96, 410;
+97, 405;
+98, 400;
+99, 395;
+100, 390
+```
+
+# Start (ignition) time to N
+
+```
+second, N
+0, 0;
+1, 4;
+2, 11;
+3, 14;
+4, 17;
+5, 20;
+6, 22;
+7, 23;
+8, 26;
+9, 27;
+10, 29;
+11, 30;
+12, 32;
+13, 33;
+14, 35;
+15, 36;
+16, 37;
+17, 39;
+18, 41;
+19, 43;
+20, 45;
+21, 47;
+22, 49;
+23, 52;
+24, 54;
+25, 56;
+26, 58;
+27, 60;
+28, 62;
+29, 65;
+30, 67;
+31, 69;
+32, 72;
+33, 74;
+34, 75;
+35, 77;
+36, 79;
+37, 81;
+38, 83;
+39, 85;
+40, 88;
+41, 91;
+42, 93;
+43, 96;
+44, 99;
+45, 100
+```
+
+# N to Volt
+
+```
+N, V
+0-83, 0;
+84, 105;
+85, 115;
+86, 115;
+87, 115;
+88, 114;
+89, 115;
+90, 115;
+91, 114;
+92, 115;
+93, 115;
+94, 115;
+95, 115;
+96, 115;
+97, 115;
+98, 115;
+99, 114;
```

Comment:
The variation in voltage at different speeds here and at N=88 are artifacts of some external change in load.

The golden rules of AC power flow are:
- voltage is strongly related to reactive power
- frequency and phase angle are strongly related to real power
  - in the case of a generator also the power input as a result of the prime mover and governors ability to match prime mover output to load => maintain frequency

We can model the electrical system in more detail at a later date and make voltages, frequencies, and currents more realistic/less constant.